jenkins/runlocal

```diff
@@ -1,29 +1,26 @@
 #!/bin/bash -e
 
-# Build the RPM and then build the containers, pushing only the CI image
-# to the registry, and then run the fuctional tests against the built
-# container image.
+# Build the pbench-server RPM locally, then build the containers locally, and
+# then run the fuctional tests against the locally built CI container image.
 
-export PB_CONTAINER_REG=images.paas.redhat.com
-export PB_ORG_NAME=pbench
+export PB_SERVER_CONTAINER_REG=localhost
 export PB_SERVER_IMAGE_NAME=pbench-server-ci
 export PB_SERVER_IMAGE_TAG=${USER}
-export PB_DASHBOARD_DIR=$(pwd)/dashboard/build
```

Comment:
Although we're not now, we will be adding UI functional tests. Should we really be dropping this?

Reply:
I don't think that `PB_DASHBOARD_DIR` is referenced by this script or by anything which it invokes, except for `server/pbenchinacan/run-pbench-in-a-can` which already provides the same value if it's not already defined.